pypdf/_page.py

```diff
@@ -1980,12 +1980,21 @@ def process_operation(operator: bytes, operands: List[Any]) -> None:
                 memo_tm = tm_matrix.copy()
             # Table 5.2 page 398
             elif operator == b"Tz":
-                char_scale = float(operands[0]) / 100.0
+                if operands:
+                    char_scale = float(operands[0]) / 100.0
+                else:
+                    char_scale = 1.0
```

Comment:
```suggestion
                char_scale = float(operands[0]) / 100.0 if operands else 1.0
```

I propose this to prevent coverage issue. and this looks still readable

Reply:
I am not sure about this - this just hides that some parts have no coverage.

Reply:
Remove the conditions for the untested cases and wait until someone reports them as well.

Reply:
I prefer my proposal 😳